iotedgedev/modules.py

```diff
@@ -145,7 +145,7 @@ def build_push(self, template_file, default_platform, no_build=False, no_push=Fa
                     # get the Dockerfile from module.json
                     dockerfile = module.get_dockerfile_by_platform(platform)
                     container_tag = "" if self.envvars.CONTAINER_TAG == "" else "-" + self.envvars.CONTAINER_TAG
-                    tag = "{0}:{1}{2}-{3}".format(module.repository, module.tag_version, container_tag, platform).lower()
+                    tag = "{0}:{1}{2}-{3}".format(module.repository, module.tag_version, container_tag, platform)
```

Comment:
FYI for future reviewers...this comment can be closed.

Image name needs to be lower case.  module.repository = {server}/{name}, so you need to call .lower on that, but not on the tag.  I have confirmed that name.lower is called earlier in the code.